config/puma.rb

```diff
@@ -6,6 +6,7 @@
 #
 threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
 threads threads_count, threads_count
+workers_count = ENV.fetch('WEB_CONCURRENCY') { 1 }
```

Comment:
__[RuboCop]__

Lint/UselessAssignment: Useless assignment to variable - `workers_count`.


<!-- Comment by [Sider](https://sider.review) -->